common/changes/@microsoft/rush/master_2019-06-06-23-07.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "comment": "Fix git process in rush version",
+      "packageName": "@microsoft/rush",
+      "type": "none"
+    }
+  ],
+  "packageName": "@microsoft/rush",
+  "email": "dahajek@microsoft.com"
```

Comment:
Update this to use `dahajek@users.noreply.github.com` per our [repo policy](https://github.com/microsoft/web-build-tools/blob/df9eaf65a4f0c378c2266ce1a66aedd8b58e4bb5/rush.json#L168)

Reply:
Updated.